cli/dcoscli/main.py

```diff
@@ -137,3 +138,24 @@ def signal_handler(signal, frame):
     emitter.publish(
         errors.DefaultError("User interrupted command with Ctrl-C"))
     sys.exit(0)
+
+
+def handle_docopt_error(func):
```

Comment:
Rename this function - it's not handling the error, more like decorating it. Perhaps `def decorate_docopt_usage(func)`